util/progress/progressui/colors.go

```diff
@@ -0,0 +1,60 @@
+package progressui
+
+import (
+	"fmt"
+	"regexp"
+	"strings"
+
+	"github.com/morikuni/aec"
+)
+
+var rex = regexp.MustCompile("(\\w+)=(\\w+)")
+
+var termColorMap = map[string]aec.ANSI{
+	"default": aec.DefaultF,
+
+	"black":   aec.BlackF,
+	"blue":    aec.BlueF,
+	"cyan":    aec.CyanF,
+	"green":   aec.GreenF,
+	"magenta": aec.MagentaF,
+	"orange":  aec.Color8BitF(aec.NewRGB8Bit(255, 140, 0)),
+	"red":     aec.RedF,
+	"white":   aec.WhiteF,
+	"yellow":  aec.YellowF,
+
+	"light-black":   aec.LightBlackF,
+	"light-blue":    aec.LightBlueF,
+	"light-cyan":    aec.LightCyanF,
+	"light-green":   aec.LightGreenF,
+	"light-magenta": aec.LightMagentaF,
+	"light-orange":  aec.Color8BitF(aec.NewRGB8Bit(255, 165, 0)),
+	"light-red":     aec.LightRedF,
+	"light-white":   aec.LightWhiteF,
+	"light-yellow":  aec.LightYellowF,
+}
+
+func setUserDefinedTermColors(colorsEnv string) {
+
+	data := rex.FindAllStringSubmatch(colorsEnv, -1)
+	for _, kv := range data {
+		k := kv[1]
+		v := kv[2]
+		// debug
+		fmt.Print(k, "=", v)
+		c, ok := termColorMap[strings.ToLower(v)]
```

Comment:
I liked the human-friendly color names for ease of use, but might also be good to support the numbered colors like `LS_COLORS`, for more options?

Reply:
Done